src/page-attributes.ts

```diff
@@ -36,7 +36,26 @@ function readPageAttributes() {
     throw new Error('"origin" is required.');
   }
 
-  const matches = repoRegex.exec(params.repo);
+  let repo = params.repo;
+  let repoParts = repo.split('/');
+
+  if (repoParts.length >= 2) {
+    let repoOwner = repoParts[repoParts.length - 2];
+    let colonIndex = repoOwner.lastIndexOf(':');
```

Comment:
"colonIndex" should be declared with "const" too.

Reply:
Agreed. Thanks for reviewing.